packages/wallet/test/services/util.test.ts

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { calculateStabilityWindowSlotsCount } from '../../src/util';
+
+describe('calculateStabilityWindowSlotsCount', () => {
+  it('computes stability window slots count', () => {
+    const securityParameter = 2160;
+    const activeSlotsCoefficient = 0.05;
+    const expectedStabilityWindowValue = 129_600;
+    expect(calculateStabilityWindowSlotsCount({ activeSlotsCoefficient, securityParameter } as any)).toEqual(
```

Comment:
cast to any should not be necessary once the return type for `calculateStabilityWindowSlotsCount ` is specified

Reply:
Here I cast the function arguments (`genesis: CompactGenesis` type) to any, not the return type. Even with the return type for `calculateStabilityWindowSlotsCount` (`number`), the genesis type consists of many other props which we do not need in the test. @mirceahasegan

Reply:
Got it now. Can you try to cast to `Partial<CompactGenesis>`, instead of `any`? Then it would still check that the properties included are actually part of `CompactGenesis`.

Reply:
yep, exactly what was going to change :)

Reply:
Applied in https://github.com/input-output-hk/cardano-js-sdk/pull/433/commits/4c407e6e7521733781853cc636c544f6613584ff